source/features/forked-to.tsx

```diff
@@ -101,6 +102,8 @@ async function updateUI(forks: string[]): Promise<void> {
 			</details>,
 		);
 	}
+
+	groupButtons([forkButton, select('.rgh-forked-button')!]);
```

Comment:
I think you can alternatively just use:

```suggestion
	groupSiblings(forkButton);
```

Reply:
I had tested that before - it only worked in the case of the single button. With the dropdown, it looked really messy. The button ended up being on a new line entirely. `groupSiblings` didn't add the dropdown element to the `BtnGroup`, but it worked with the single button.

Using `groupButtons` with the fork button and the class, works for both the single button and the dropdown though